dao-contracts/src/kyc_voter.rs

```diff
@@ -93,25 +94,31 @@ impl KycVoterContractInterface for KycVoterContract {
             fn get_variable_repo_address(&self) -> Address;
             fn get_reputation_token_address(&self) -> Address;
             fn get_dust_amount(&self) -> U256;
-            fn get_voting(&self, voting_id: U256) -> Option<Voting>;
-            fn get_ballot(&self, voting_id: U256, address: Address) -> Option<Ballot>;
-            fn get_voter(&self, voting_id: U256, at: u32) -> Option<Address>;
+            fn get_voting(&self, voting_id: VotingId) -> Option<Voting>;
+            fn get_ballot(&self, voting_id: VotingId, address: Address) -> Option<Ballot>;
+            fn get_voter(&self, voting_id: VotingId, at: u32) -> Option<Address>;
         }
     }
 
-    fn create_voting(&mut self, subject_address: Address, document_hash: U256, stake: U256) {
+    fn create_voting(
+        &mut self,
+        subject_address: Address,
+        _document_hash: DocumentHash,
```

Comment:
Can't we remove it?

Reply:
We need to keep it. It will be recorded as an argument of the contract call. We'll probably end up with specific event for every `create_voting` and will use it there.